libs/SalesforceRestAPI/SalesforceRestAPI/Classes/SFRestAPI.m

```diff
@@ -31,7 +31,7 @@
 #import <SalesforceSDKCore/SFSDKWebUtils.h>
 #import <SalesforceSDKCore/SalesforceSDKManager.h>
 
-NSString* const kSFRestDefaultAPIVersion = @"v31.0";
+NSString* const kSFRestDefaultAPIVersion = @"v32.0";
```

Comment:
ContentNote are in 32 (anyway we should always point to the most recent release number)